packages/scandipwa/src/component/CategoryConfigurableAttributes/CategoryConfigurableAttributes.component.js

```diff
@@ -9,15 +9,36 @@
  * @link https://github.com/scandipwa/base-theme
  */
 
+import PropTypes from 'prop-types';
+
 import ExpandableContent from 'Component/ExpandableContent';
 import ExpandableContentShowMore from 'Component/ExpandableContentShowMore';
 import ProductAttributeValue from 'Component/ProductAttributeValue/ProductAttributeValue.component';
 // eslint-disable-next-line max-len
 import ProductConfigurableAttributes from 'Component/ProductConfigurableAttributes/ProductConfigurableAttributes.component';
+import { CategoryFragment } from 'Type/Category';
 import { formatPrice } from 'Util/Price';
 
 /** @namespace Component/CategoryConfigurableAttributes/Component */
 export class CategoryConfigurableAttributes extends ProductConfigurableAttributes {
+    static propTypes = {
+        childrenCategories: PropTypes.arrayOf(PropTypes.shape(CategoryFragment))
+    };
+
+    renderSubCategories(option) {
+        const optionWithSubcategories = option;
```

Comment:
This assignment is useless. What are you intending to achieve here?
This assignment provides an additional reference to the object which is referenced by `option` variable.
Hence further in the code, you mutate the original object. If you want to create a copy in order to respect data immutability principle of FP, use `Object.assign(...)` or `{ ... }`.